test/e2e/indexTable/indexTablePage.ts

```diff
@@ -0,0 +1,24 @@
+import { Locator, Page } from '@playwright/test';
+import { StoryBookPage } from '../storybookPage';
+
+export class IndexTablePage extends StoryBookPage {
+  [x: string]: any;
+  readonly checkboxes: Locator;
+  readonly container: Locator;
+  readonly tableRow: Locator;
+  readonly moveToAction: Locator;
+
+  constructor(page: Page) {
+    super(page, '?path=/story/next-indextable--interactive');
+    this.container = page.locator('#storybook-preview-wrapper');
+    this.checkboxes = page
+      .frameLocator('internal:attr=[title="storybook-preview-iframe"i]')
+      .getByRole('checkbox');
+    this.tableRow = page
```

Comment:
I think this one should be done as checkboxes, return list and let the test to decide which row they wanna use